contrib/utilities/indent_common.sh

```diff
@@ -66,6 +66,43 @@ checks() {
     echo "***   to install a compatible binary into './contrib/utilities/programs'."
     exit 1
   fi
+
+
+  # check formatting of usernames and email addresses, examples that will be detected:
+  # not-using-a-name <a@b.com>
+  # John Doe <doe@macbook.local>
+  # Jane Doe <a@nodomain>
+  #
+  # For commits already in the history, please see .mailmap in the root directory.
+  #
+  # Note that we currently allow email addresses of the form
+  # Luca Heltai <luca-heltai@users.noreply.github.com>
+  # as these are generated when using the website to commit.
+  #
+  # Finally, to stay sane, just go back until the beginning of 2019 for now.
+  #
+  # first user names:
+  git log --since "2019-01-01" --format="%aN" | sort -u | while read name ; do
```

Comment:
Shouldn't we do this `since master` (or whatever the syntax is)?

Reply:
I don't understand how this works, but it does only seem to complain about newer commits.

Reply:
I don't want to apply fixes in .mailmap for our whole history for now. We can work on that as a follow-up if you feel strongly about it. Yes, we could make a complicated logic to only check the commits not in master, but this is much easier.

Reply:
Sorry, I should have been clearer in my second commit: this works correctly so I am OK with the current implementation.